src/tests/data/F2F_YOTI_START_00_SCHEMA.json

```diff
@@ -1,143 +1,207 @@
+          },
+          "required": [
+              "evidence",
+              "post_office_details"
+          ]
+      },
+      "restricted": {
+          "type": "object",
+          "properties": {
+              "name": {
+                  "type": "array",
+                  "items": [
+                      {
+                          "type": "object",
+                          "properties": {
+                              "nameParts": {
+                                  "type": "array",
+                                  "items": [
+                                      {
+                                          "type": "object",
+                                          "properties": {
+                                              "type": {
+                                                  "type": "string"
+                                              },
+                                              "value": {
+                                                  "type": "string"
+                                              }
+                                          },
+                                          "required": [
+                                              "type",
+                                              "value"
+                                          ]
+                                      },
+                                      {
+                                          "type": "object",
+                                          "properties": {
+                                              "type": {
+                                                  "type": "string"
+                                              },
+                                              "value": {
+                                                  "type": "string"
+                                              }
+                                          },
+                                          "required": [
+                                              "type",
+                                              "value"
+                                          ]
+                                      },
+                                      {
+                                          "type": "object",
+                                          "properties": {
+                                              "type": {
+                                                  "type": "string"
+                                              },
+                                              "value": {
+                                                  "type": "string"
+                                              }
+                                          },
+                                          "required": [
+                                              "type",
+                                              "value"
+                                          ]
+                                      }
```

Comment:
Out of interest why do we need the same object type defined 3 times in the array?

```
                                      {
                                          "type": "object",
                                          "properties": {
                                              "type": {
                                                  "type": "string"
                                              },
                                              "value": {
                                                  "type": "string"
                                              }
                                          },
                                          "required": [
                                              "type",
                                              "value"
                                          ]
                                      }
```

Reply:
I think it's because we have 3 different objects in the nameParts block in the F2F_YOTI_START event. 

![image](https://github.com/govuk-one-login/ipv-cri-f2f-api/assets/110032361/ffe37cdf-42a7-4b5f-8324-06b95d969c24)